langchain/llms/llamacpp.py

```diff
@@ -282,3 +287,30 @@ def stream(
                     token=token, verbose=self.verbose, log_probs=log_probs
                 )
             yield chunk
+
+    def _get_max_tokens(self, prompt: str) -> int:
```

Comment:
could we add a unit test for this? will probably want to make this a standalone function that takes a Llama client as a arg